security/publish_dispatcher_spec.rb

```diff
@@ -17,8 +17,23 @@
   end
 
   it 'should not be able to access Publish pages as site visitor' do
+    secure_urls = File.readlines('security/secure_public_routes.txt')
+    # check each url
+    secure_urls.each do |url|
+      visit url
+      # check if its a redirect
```

Comment:
Rephrase this to: follow redirection since Capybara doesn't do this automatically